src/bitfield/query.py

```diff
@@ -27,12 +27,10 @@ def as_sql(self, qn, connection=None):
 
     class BitQueryLookupWrapper(Exact):  # NOQA
         def process_lhs(self, qn, connection, lhs=None):
-            lhs_sql, params = super(BitQueryLookupWrapper, self).process_lhs(qn, connection, lhs)
-            if self.rhs:
-                lhs_sql = lhs_sql + ' & %s'
-            else:
-                lhs_sql = lhs_sql + ' | %s'
-            params.extend(self.get_db_prep_lookup(self.rhs, connection)[1])
+            lhs_sql, params = super(BitQueryLookupWrapper, self).process_lhs(
```

Comment:
im pretty confused about this and cant understand how it works by skimming django source code

also all forks of bitfield I found use our old code, which doesn't seem to work correctly at least in Django 1.8

im fully relying on our test suites to prove this isnt completely wrong